src/ducks/sharing/index.js

```diff
@@ -38,6 +38,44 @@ export const createPermSet = (id, doctype) =>
     }
   })
 
+export const share = async (id, doctype, email, url) => {
+  const target = await cozy.client.fetchJSON('POST', '/sharings/recipient', {
+    email,
+    url
+  })
+
+  const sharings = await cozy.client.fetchJSON('POST', '/sharings/', {
+    desc: 'Share Ilyn Sarkisian',
```

Comment:
The `/sharings` route accepts a `desc` property to add what I think to be a description (maybe for the email body sent to the recipient). As I don't know what to add, and as it is related to share, I though [her name](https://en.wikipedia.org/wiki/Cher) could be a good description.